app/tools/__init__.py

```
@@ -1,2 +1,3 @@
 from .google_sheets_manager import GoogleSheetsManager
 from .news_scraper import NewsScraper
+from .social_media_scraper import SocialMediaScraper
```

Ruff (0.12.2)

`.social_media_scraper.SocialMediaScraper` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)